doc/nrf/ug_thingy53.rst

```diff
@@ -285,3 +260,104 @@ To build and program the source code from the command line, complete the followi
+Thingy:53-compatible applications can be updated using DFU functionality with either `nRF Connect Programmer`_ or nRF Programmer mobile application.
+
+Enabling DFU and MCUboot requires consistency in configuration of the samples and applications.
+For a sample or application to be compatible with Thingy:53, the application must comply with the following configuration:
+
+Partition manager configuration
+  Thingy:53 compatible samples use :ref:`partition_manager` to define memory partitions.
+  To ensure that partition layout does not change between builds, the sample must use static partition layout that is the same as the one in Thingy:53-compatible samples in the |NCS|.
+  The memory layout must stay consistent, so that MCUboot can perform proper image updates and clean up the settings storage partition.
+
+  The PCD SRAM partition is locked by the MCUboot bootloader to prevent the application from modifying the network core firmware.
+  Trying to access data on this partition results in an ARM fault.
+
+MCUboot bootloader
+  The sample must enable MCUboot bootloader and use the same configuration of the bootloader as all of the Thingy:53-compatible samples in the |NCS|.
+  This is needed to let MCUboot use the sample as an image update.
+
+  The MCUboot bootloader supports serial recovery and a custom command to erase settings storage partition.
+  Erasing settings parition is needed to ensure that an application would not be booted with incompatible content of settings partition.
+
+In addition to the configurations mentioned above, an application compatible with Thingy:53 is expected to enable the following features:
+
+* USB CDC ACM as backend for logger.
+  The logs are provided via USB CDC ACM to allow accessing them without additional hardware.
+
+  Most of the Thingy:53-compatible applications and samples use only a single instance of USB CDC ACM that works as logger's backend.
+  No other USB classes are used.
+  These samples can share common USB product name, vendor ID, and product ID.
+  If a sample supports additional USB classes or more than one instance of USB CDC ACM, it must use dedicated product name, vendor ID, and product ID.
+
+* DFU over-the-air via Simple Management Protocol over Bluetooth.
+  The application acts as GATT server and allows the connected Bluetooth Central to perform the firmware update for both application and network core.
+  The Bluetooth configuration is updated to allow quick DFU data transfer.
+  The application supports Simple Management Protocol (SMP) handlers related to:
+
+  * Image management
+  * Operating System (OS) management -- used to reboot device after firmware upload is complete.
+  * Erasing settings partition -- needed to ensure that a new application would not be booted with incompatible content of settings partition that was written by the previous application.
```

Comment:
Erasing settings partition -- needed to ensure that a new application would not be booted with incompatible content in the settings partition that was written by the previous application.

Reply:
changed